walle_rl/buffer/buffer.py

```diff
@@ -117,10 +118,15 @@ def store(self, **kwargs):
             self.full = True
             self.ptr = 0
 
-    def _get_batch_by_ids(self, batch_ids, env_ids):
+
+    @partial(jax.jit, static_argnames=['self'])
```

Comment:
this one line amazingly sped up the update part by ~4x. (and this function itself likely much higher.)